### PR TITLE
[#800] Metadata Subfield Sizes

### LINKED ISSUE #800: Export Metadata Subfield Sizes
Export the Metadata subfield [Sizes](https://inveniordm.docs.cern.ch/reference/metadata/#sizes-0-n)

### PR DESCRIPTION
**Pull request by Meowcenary**
Add sizes subfield to metadata function. Only page counts are included
at this time. closes #800

### CODE REVIEW

File: spec/models/invenio_rdm_record_converter_spec.rb [hunk 1]
```diff
@@ -28,7 +28,8 @@
       date_created: ["1-1-2021"],
       mime_type: 'application/pdf',
       grants_and_funding: ["European Commission 00k4n6c32"],
-      language: ["English"]
+      language: ["English"],
+      page_count: [rand(1..1000).to_s]
```
Comment (Meowcenary) [inline spec/models/invenio_rdm_record_converter_spec.rb:32]: Randomize the page count

File: app/models/invenio_rdm_record_converter.rb [hunk 2]
```diff
@@ -119,6 +119,7 @@ def invenio_metadata(gf)
       "subjects": SUBJECT_SCHEMES.map{ |subject_type| subjects_for_scheme(gf.send(subject_type), subject_type) }.flatten,
       "dates": gf.date_created.map{ |date| {"date": date, "type": "other", "description": "When the item was originally created."} },
       "languages": gf.language.any?{ |lang| lang.downcase == ENGLISH} ? ["eng"] : "",
+      "sizes": Array.new.tap{ |size_json| size_json << "#{gf.page_count} pages" if !gf.page_count.blank? },
```
Comment (Meowcenary) [inline app/models/invenio_rdm_record_converter.rb:122]: Tap is just the greatest for this kind of thing.

**Comment by Meowcenary** [comment]
Very small and simple PR. File size is no longer included since Invenio will calculate file size itself.

**Review by sharpattack — COMMENTED** [review]
Looks good to me. Just curious though, why only `pages`? Do you have an example record with this, I can't seem to find one. Any other "size" info we could use?

**Comment by Meowcenary** [comment]
Here is the [documentation on Sizes](https://inveniordm.docs.cern.ch/reference/metadata/#sizes-0-n) if you would like to see an official example. File size is not being used because, per @fenekku's feedback, it will be calculated by the database. The sufia model only had page counts and byte size so it seemed that page count was the only option that could go in. Also FWIW, this is an optional field so I think _anything_ we put in is just details.

**Comment by fenekku** [comment]
On top of my head, the only other "sizes" is duration when the resource is a movie/audio clip, but don't worry about it.

**Review by sharpattack — APPROVED** [review]
Found an example in DigitalHub. I was only seeing "pages" on Collections, and thought there may not be any on GenericFiles. Looks good to go.

**Pull request merged**